index.html

```diff
@@ -0,0 +1,338 @@
+ 
+                     <li><a href="#" class="fa fa-angellist"></a></li>
+                    </ul>
+                  </div>
+            <div class="card-button ">
+              <button type="button" class="button1">Get my resume</button>
+            </div>
+                <ul>
+                  <li>
+            <div class="progrm-stack ">
+              <h3 class="Aboutme-subtitle ">Languages</h3>
+              <div class="progrm-stack1"></div>
+              <div class="progrm-stack-item">
+                <div class="arrow-img ">
+                  <img src="./img/arrow-down.png" class="arrow-image" alt=""  />
+                  </div>
+              </div>
+            </div>
+        </li>
+              <li>
+            <div class="all-stack ">
+              <div class="stack-body ">
+                <div class="stack-body-item">
+                  <div class="stack-icon ">
+                   <img src="./img/Javascript.png" alt="" width="50" height="50"/>
+                       </div>
+                </div>
+                <div class="stack-body-s"></div>
+                <h5 class="stack-type ">JavaScript</h5>
+              </div>
+            </div>
+            </li>
+            <li>
+            <div class="all-stack ">
+              <div class="stack-body ">
+                <div class="stack-body-item">
+                  <div class="stack-icon ">
+                   <img src="./img/Html5.png" alt="" width="50" height="50"/>
+                       </div>
+                </div>
+                <div class="stack-body-s"></div>
+                <h5 class="stack-type ">HTML</h5>
+              </div>
+            </div>
+            </li>
+            <li>
+            <div class="all-stack ">
+              <div class="stack-body ">
+                <div class="stack-body-item">
+                  <div class="stack-icon ">
+                   <img src="./img/css3.png" alt="" width="50" height="50"/>
+                       </div>
+                </div>
+                <div class="stack-body-s"></div>
+                <h5 class="stack-type ">CSS</h5>
+              </div>
+            </div>
+            </li>
+            </ul>
+          
+            <div class="framework ">
+              <h3 class="Aboutme-subtitle ">Frameworks</h3>
+              <div class="framework-spacer"></div>
+              <div class="framework-item">
+                <div class="Aboutme-image ">
+                 <img src="./img/arrow-forward.png" alt="">
+                    </div>
+              </div>
+            </div>
+            <hr class="Abt-border " />
+            <div class="framework">
+              <h3 class="Aboutme-subtitle ">Skills</h3>
+              <div class="framework-spacer1"></div>
+              <div class="framework-item">
+                <div class="Aboutme-image5 ">
+                  <img src="./img/arrow-forward.png" alt="">
+                    </div>
```

Comment:
- [ ] Kindly make the design similar to the Figma design. Make changes to those sections that I point out. Make similar to Figma design
### your design
![his](https://user-images.githubusercontent.com/81727988/170823326-e8045d87-8b7a-45ab-8805-d77b7dc7496c.png)